src/cider/nrepl/middleware/classpath.clj

```diff
@@ -1,16 +1,26 @@
 (ns cider.nrepl.middleware.classpath
   (:require [cider.nrepl.middleware.util.error-handling :refer [with-safe-transport]]
-            [orchard.misc :as u]
             [clojure.java.classpath :as cp]
-            [clojure.string :as str]))
+            [clojure.string :as str]
+            [orchard.misc :as u])
+  (:import [java.io File]))
+
+(def java-major-version
```

Comment:
We already have a function for this - `jdk-api-version`.

Reply:
Where? I can find no such function in the project.